tests_smoke/.env.example

```diff
@@ -1,15 +1,16 @@
-API_HOST_NAME=
+SMOKE_API_HOST_NAME=
+SMOKE_POLL_TIMEOUT=
 
-AWS_ACCESS_KEY_ID=
-AWS_SECRET_ACCESS_KEY=
-CSV_UPLOAD_BUCKET_NAME=
+SMOKE_AWS_ACCESS_KEY_ID=
+SMOKE_AWS_SECRET_ACCESS_KEY=
+SMOKE_CSV_UPLOAD_BUCKET_NAME=
 
-ADMIN_CLIENT_SECRET=
+SMOKE_ADMIN_CLIENT_SECRET=
 
-EMAIL_TO=
-SMS_TO=
-USER_ID=
-SERVICE_ID=
-SMS_TEMPLATE_ID=
-EMAIL_TEMPLATE_ID=
-API_KEY=
+SMOKE_EMAIL_TO=
+SMOKE_SMS_TO=
+SMOKE_USER_ID=
+SMOKE_SERVICE_ID=
+SMOKE_SMS_TEMPLATE_ID=
+SMOKE_EMAIL_TEMPLATE_ID=
+SMOKE_API_KEY=
```

Comment:
I changed the names to make it easier in the manifests repo to show which secrets were for the smoke tests and which were for the other stuff.